magefile.go

```diff
@@ -158,6 +165,27 @@ func copyFile(f *zip.File, dstPath string) error {
 	return nil
 }
 
+func cleanupOldCRS(rulesDstDir, testsDir string) error {
+	if err := os.RemoveAll(rulesDstDir); err != nil {
+		return err
+	}
+	if err := filepath.WalkDir(testsDir, func(path string, d os.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if strings.HasPrefix(d.Name(), "REQUEST-") || strings.HasPrefix(d.Name(), "RESPONSE-") {
```

Comment:
Wonder about this, shall we just remove everything? Otherwise could you add a comment on why this condition?

Reply:
Refactored to make it more explicit. The goal is to do not delete .go files that belongs to this repo

Reply:
I am lost now, do we iterate over root files in this repo? Can't we just delete `rules` folder and `tests` folder?